jsonate/json_encoder.py

```diff
@@ -36,6 +37,14 @@ def default(self, obj):
             return super(JsonateEncoder, self).default(obj)
         except TypeError:
             return obj
+
+
+class UUIDEncoder(JsonateEncoder):
+    def default(self, obj):
+        if isinstance(obj, UUID):
+            # if the obj is uuid, we simply return the value of uuid
+            return obj.hex
+        return JsonateEncoder.default(self, obj)
```

Comment:
This works, but it's not inline with the existing serializations. 

Check examples at the bottom of this file to see how `FieldFile`, `decimal` and other types are serialized.